frontend/src/metabase/query_builder/components/NativeQueryEditor.jsx

```diff
@@ -67,11 +67,16 @@ class NativeQueryEditor extends Component {
   static defaultProps = {
     isOpen: false,
     cancelQueryOnLeave: true,
+    resizable: true,
   };
 
   UNSAFE_componentWillMount() {
     const { question, setIsNativeEditorOpen, isInitiallyOpen } = this.props;
-    setIsNativeEditorOpen(!question || !question.isSaved() || isInitiallyOpen);
+    if (setIsNativeEditorOpen) {
```

Comment:
this and similar conditionals are necessary because these are now optional props that the action editor doesn't care about (like collapsing the native editor)